pkg/alpine/Dockerfile

```diff
@@ -1,7 +1,9 @@
-ARG EVE_BUILDER_IMAGE=lfedge/eve-alpine:6.7.0
+ARG EVE_BUILDER_IMAGE=alpine:3.16.2
```

Comment:
Just to make sure I understand the flow, once we have build a new lfedge/eve-alpine using this PR, will we change this line to refer to lfedge/eve-alpine:<version>? 
I assume we need that to ensure reproducible builds.

Reply:
Correct, also I will add new-kernel, acrn and acrn-kernel updates (with them we are not fit CI timeout for arm64).